core/src/templates/template.html.xslt.js

```diff
@@ -135,6 +135,11 @@ export default `<?xml version="1.0" encoding="utf-8" standalone="yes"?>
     <xsl:value-of select="concat($dd,'.',$mm,'.',$yyyy)"/>
   </xsl:template>
 
+  <xsl:template name="base_format_time">
+    <xsl:param name="instr"/>
+    <xsl:value-of select="format-time($instr, '[H01]:[m01]')"/>
```

Comment:
Format time with leading zero for now to be backward compatible with simple string.